src/main/java/org/graylog/plugins/collector/CollectorMetaData.java

```diff
@@ -47,7 +44,7 @@ public String getAuthor() {
 
     @Override
     public URI getURL() {
-        return URI.create("https://www.graylog.org/");
+        return URI.create("http://docs.graylog.org/en/latest/pages/pipelines.html");
```

Comment:
This link does not go to the collector docs. :wink:

Reply:
It's the kind of pipeline collector…

Yeah, good catch and bad copy & paste. 😉